server/common/nosql/mongo-dao/src/test/java/org/kaaproject/kaa/server/common/nosql/mongo/dao/CredentialsMongoDaoTest.java

```diff
@@ -0,0 +1,46 @@
+package org.kaaproject.kaa.server.common.nosql.mongo.dao;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kaaproject.kaa.common.dto.credentials.CredentialsDto;
+import org.kaaproject.kaa.server.common.dao.model.Credentials;
+import org.kaaproject.kaa.server.common.nosql.mongo.dao.model.MongoCredentials;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.IOException;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = "/mongo-dao-test-context.xml")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+public class CredentialsMongoDaoTest extends AbstractMongoTest {
+    private static final Logger LOG = LoggerFactory.getLogger(CredentialsMongoDaoTest.class);
+    @BeforeClass
+    public static void init() throws Exception {
+        MongoDBTestRunner.setUp();
+    }
+
+    @AfterClass
+    public static void after() throws Exception {
+        MongoDBTestRunner.tearDown();
+    }
+
+    @After
+    public void afterTest() throws IOException {
+        MongoDataLoader.clearDBData();
+    }
+
+    @Test
+    public void findCredentialByIdTest() {
+        String id = "56f3f9acdd796a86323f04a4";
+        Credentials credentials = credentialsDao.findById(id);
+        CredentialsDto credentialsDto = credentials.toDto();
+        System.out.println(credentialsDto);
```

Comment:
You do not use print statements to test things using JUnit. You use the [Assert](http://junit.sourceforge.net/javadoc/org/junit/Assert.html) class.